src/main/java/polyglot/ServiceList.java

```diff
@@ -0,0 +1,108 @@
+package polyglot;
+
+import java.io.File;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import com.google.common.base.Joiner;
+import com.google.protobuf.DescriptorProtos.FileDescriptorSet;
+import com.google.protobuf.Descriptors.Descriptor;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+import com.google.protobuf.Descriptors.MethodDescriptor;
+import com.google.protobuf.Descriptors.ServiceDescriptor;
+
+import polyglot.protobuf.ServiceResolver;
+
+/** Simple utility to list the services, methods and message definitions for the known GRPC end-points */
+public class ServiceList {
```

Comment:
A very simple unit test would be nice, to make sure future changes don't break this.